banman/store.go

```diff
@@ -66,6 +66,9 @@ type Store interface {
 
 	// Status returns the ban status for a given IP network.
 	Status(*net.IPNet) (Status, error)
+
+	// UnBanIPNet unbans previously banned peer
+	UnBanIPNet(ipNet *net.IPNet) error
```

Comment:
nit: `unbanIPNet`? I am not an english speaker but I think it is the same as with the word "unable"?

Reply:
yes, I meant capitalizing only the first letter `UnbanIPNet` instead of `UnBanIPNet`. Like [here](https://github.com/bitcoin/bitcoin/blob/3132ec64d9e0f7af88e61ac35807f44315a5ca96/src/banman.h#L77)

Reply:
I have effected the change @positiveblue